README-KO.md

```diff
@@ -842,6 +844,25 @@ def getAddress(name: String): Option[String] = {
 
 ```
 
+다른 예로는 if-else가 전체 if-else의 모나드 체인인지 아니면 else 절의 모나드 체인인지 햇갈리는 경우가 있습니다.
+```scala
+// A monadic chaining approach
+val condition: Boolean = ...
+if (condition) {
+  Seq(1, 2, 3)  // Results in List(2, 3, 4)
+} else {
+  Seq(1, 2, 3)  // Results in List(1, 2, 3)
+}.map(_ + 1)
```

Comment:
Wait, where the monad chain applies? I have been thinking that it applies to the entire if-else, but if it doesn't, it should be else, then code comment seems to be swapped.

Reply:
uh oops